pandas/tests/series/indexing/test_indexing.py

```diff
@@ -907,3 +907,13 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+@pytest.mark.parametrize("type_", ["uint8", "uint16", "uint32", "uint64"])
```

Comment:
can you use the uint_dtype fixture (literally drop the parametrize), and pass uint_dtype as the arg (and rename type_ -> uint_dtype), ping on green.

Reply:
actually might as well use 'any_int_dtype' here